internal/transport/http2_client.go

```diff
@@ -388,6 +395,12 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 			Val: *opts.MaxHeaderListSize,
 		})
 	}
+	if opts.MaxFrameSize != nil {
+		ss = append(ss, http2.Setting{
+			ID:  http2.SettingMaxFrameSize,
+			Val: *opts.MaxFrameSize,
```

Comment:
Use `maxFrameSize` here since it will be sanitized.

Reply:
done